src/picasso-definition/interactions/pan.js

```diff
@@ -48,20 +58,31 @@ const pan = ({ chart, actions, viewHandler, rtl }) => ({
       e.preventDefault();
       viewHandler.setInteractionInProgress(true);
       this.started = eventName;
+
+      // A pan always starts with a tap/click
+      // Therefore, the tap/click has to be handled first
+      let panInMiniChart = false;
```

Comment:
This part seems wrong. The pan should not have to know about the tap, so there should not be any need for tap related code in pan interaction.

What problem will this solve?

I might have this wrong, but if the tap interaction is registered before the pan interaction, the tap event should prevent other events from triggering (can't remember if additional config is needed). Is this the problem this is trying to solve?

Reply:
Is this still needed?